cookbooks/fb_zfs/recipes/default.rb

```diff
@@ -0,0 +1,52 @@
+#
+# Cookbook Name:: fb_zfs
+# Recipe:: default
+#
+# vim: syntax=ruby:expandtab:shiftwidth=2:softtabstop=2:tabstop=2
+#
+# Copyright (c) 2016-present, Facebook, Inc.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree. An additional grant
+# of patent rights can be found in the PATENTS file in the same directory.
+#
+
+package 'zfs packages' do
+  package_name %w{spl spl-dkms zfs-dkms zfsutils}
+  action :upgrade
+end
+
+services = []
+if node.systemd?
+  services += %w{zfs-import-cache zfs-import-scan}
+else
+  services << 'zfs-import'
+end
+services << 'zfs-zed'
+
+services.each do |svc|
```

Comment:
needs enable/disable too... but I don't know if it's safe or not to just stop of course...